WcaOnRails/app/controllers/polls_controller.rb

```diff
@@ -42,11 +42,11 @@ def edit
   def update
     @poll = Poll.find(params[:id])
     if @poll.update_attributes(poll_params)
-      if params[:commit] == "Confirm"
-        flash[:success] = "Poll confirmed and open to voting"
-      else
-        flash[:success] = "Updated poll"
-      end
+      flash[:success] = if params[:commit] == "Confirm"
+                          "Poll confirmed and open to voting"
+                        else
+                          "Updated poll"
+                        end
```

Comment:
Yeah, "when in doubt, turn off the rule" is my approach.

There is a lot of air between "A is usually better than B" and "B should be forbidden". An informal team standard that you can break when it makes sense is my ideal.